dotcom-rendering/docs/architecture/027-pictures.md

```diff
@@ -0,0 +1,133 @@
+
+For Example:
+
+```html
+<picture>
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=620&;quality=45&;auto=format&;fit=max&;dpr=2&; 1240w" sizes="620px" media="(min-width: 980px) and (-webkit-min-device-pixel-ratio: 1.25), (min-width: 980px) and (min-resolution: 120dpi)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=620&;quality=85&;auto=format&;fit=max&; 620w" sizes="620px" media="(min-width: 980px)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=700&;quality=45&;auto=format&;fit=max&;dpr=2&; 1400w" sizes="680px" media="(min-width: 740px) and (-webkit-min-device-pixel-ratio: 1.25), (min-width: 740px) and (min-resolution: 120dpi)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=700&;quality=85&;auto=format&;fit=max&; 700w" sizes="680px" media="(min-width: 740px)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=620&;quality=45&;auto=format&;fit=max&;dpr=2&; 1240w" sizes="620px" media="(min-width: 660px) and (-webkit-min-device-pixel-ratio: 1.25), (min-width: 660px) and (min-resolution: 120dpi)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=620&;quality=85&;auto=format&;fit=max&; 620w" sizes="620px" media="(min-width: 660px)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=620&;quality=45&;auto=format&;fit=max&;dpr=2&; 1240w" sizes="480px" media="(min-width: 480px) and (-webkit-min-device-pixel-ratio: 1.25), (min-width: 480px) and (min-resolution: 120dpi)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=620&;quality=85&;auto=format&;fit=max&; 620w" sizes="480px" media="(min-width: 480px)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=465&;quality=45&;auto=format&;fit=max&;dpr=2&; 930w" sizes="375px" media="(min-width: 375px) and (-webkit-min-device-pixel-ratio: 1.25), (min-width: 375px) and (min-resolution: 120dpi)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=465&;quality=85&;auto=format&;fit=max&; 465w" sizes="375px" media="(min-width: 375px)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=465&;quality=45&;auto=format&;fit=max&;dpr=2&; 930w" sizes="320px" media="(min-width: 320px) and (-webkit-min-device-pixel-ratio: 1.25), (min-width: 320px) and (min-resolution: 120dpi)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=465&;quality=85&;auto=format&;fit=max&; 465w" sizes="320px" media="(min-width: 320px)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=465&;quality=45&;auto=format&;fit=max&;dpr=2&; 930w" sizes="0px" media="(min-width: 0px) and (-webkit-min-device-pixel-ratio: 1.25), (min-width: 0px) and (min-resolution: 120dpi)">
+    <source srcset="https://i.guim.co.uk/img/media/picture.jpg?width=465&;quality=85&;auto=format&;fit=max&; 465w" sizes="0px" media="(min-width: 0px)">
```

Comment:
In these instances, we have a `size` value that’s ignored, and often not a ratio of the `w` present in the `srcset`. It’s also worth noting that sizes refer to the [_intrinsic size_][] of the image, not its CSS pixel size.

[_intrinsic size_]: https://developer.mozilla.org/en-US/docs/Glossary/Intrinsic_Size